tests/test_file_upload.py

```diff
@@ -0,0 +1,40 @@
+from pages.file_upload import FileUploadPage
+
+# Test Case:
+# Given on the File Upload page
+# When uploading a valid file
+# Then successful file upload result
+
+
+def test_file_upload_successful(page):
+    file_upload_page = FileUploadPage(page)
+
+    # Given on the Add/Remove Elements page
+    file_upload_page.visit()
+
+    # When uploading a valid file
+    file_upload_page.choose_file("Test.txt")
```

Comment:
The name of the test file being used should be stored in a variable, for maintenance purposes

Reply:
Done